autogen/src/header.rs

```diff
@@ -34,6 +34,7 @@ fn value_enum_attribute() -> TokenStream {
 
 fn bit_enum_attribute() -> TokenStream {
     quote! {
+        #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
even though these were the original `derive` attributes with `bitflags 1`, is `(Partial)Ord` really meaningful/useful for any of these?

Reply:
Probably not. They could be added on demand at a later point without a breaking change so I think there would be nothing wrong with removing them now.

Let me know!

Reply:
Yeah, let's remove them. I double-checked all definitions and none of them seem to have any sort of meaning in ordering.

Reply:
Done.